cli/tools/installer.rs

```diff
@@ -189,7 +205,7 @@ pub fn uninstall(name: String, root: Option<PathBuf>) -> Result<(), AnyError> {
 
   // There might be some extra files to delete
   for ext in ["tsconfig.json", "lock.json"] {
-    file_path = file_path.with_extension(ext);
+    let file_path = file_path.with_extension(ext);
```

Comment:
Bug here. It would do `file.tsconfig.json` then `file.tsconfig.lock.json`. Now that's fixed.